lib/ansible/modules/cloud/azure/azure_rm_resourcegroup.py

```diff
@@ -233,7 +233,7 @@ def delete_resource_group(self):
     def resources_exist(self):
         found = False
         try:
-            response = self.rm_client.resource_groups.list_resources(self.name)
+            response = self.rm_client.resources.list_by_resource_group(self.name)
```

Comment:
we might want to catch "AttributeError" and try the old one `resource_groups.list_resources` to be ready for Azure Stack